Sparkle/SUAppcast.m

```diff
@@ -128,7 +128,7 @@ - (NSDictionary *)attributesOfNode:(NSXMLElement *)node
         if (!attrName) {
             continue;
         }
-        [dictionary setObject:[attribute stringValue] forKey:attrName];
+        [dictionary setValue:[attribute stringValue] forKey:attrName];
     }
```

Comment:
We should still use setObject:forKey: but just make sure stringValue doesn't return nil.